plugin/al/lib/AL_USDMaya/AL/usdmaya/Version.h

```diff
@@ -2,7 +2,7 @@
 // Copyright 2017 Animal Logic
 //
 // Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
+// you may not use this file except in compliance with the License.//
```

Comment:
This license change will have to be reverted.

Reply:
Yes - not sure what's adding this, will remove, sorry about that